motionplan/utils.go

```diff
@@ -10,19 +10,35 @@ import (
 	"go.viam.com/rdk/utils"
 )
 
-// FrameStepsFromRobotPath is a helper function which will extract the waypoints of a single frame from the map output of a robot path.
-func FrameStepsFromRobotPath(frameName string, path []map[string][]referenceframe.Input) ([][]referenceframe.Input, error) {
-	solution := make([][]referenceframe.Input, 0, len(path))
-	for _, step := range path {
+type Plan []map[string][]referenceframe.Input
```

Comment:
I'm adding this type in because it was a huge pain the way they were being printed in debug statements.  Now we've got a type that we can operate on and its got its own `String` function

Reply:
Yeah I'm a fan of this